event/generate.go

```diff
@@ -161,7 +161,9 @@ func main() {
 
 	fmt.Fprint(file, `//go:generate go run generate.go
 
-// Documentation source: "Event reference" by Mozilla Contributors, https://developer.mozilla.org/en-US/docs/Web/Events, licensed under CC-BY-SA 2.5.
+// Package event defines markup to bind DOM events.
+//
+// See https://developer.mozilla.org/en-US/docs/Web/Events
```

Comment:
This change will make us violate the MDN license. Let's be nice open source citizens and do proper attribution as described on https://developer.mozilla.org/en-US/docs/MDN/About#Copyrights_and_licenses